angular.json

```diff
@@ -38,14 +38,21 @@
                                 "glob": "**/*",
                                 "input": "./node_modules/ace-builds/src/",
                                 "output": "/"
+                            },
+                            {
+                                "glob": "**/*",
+                                "input": "node_modules/@taiga-ui/icons/src",
+                                "output": "assets/taiga-ui/icons"
                             }
                         ],
                         "styles": [
                             "node_modules/bootstrap/dist/css/bootstrap.min.css",
                             "node_modules/jointjs/css/layout.css",
                             "node_modules/dragula/dist/dragula.css",
                             "src/scss/material/global-material-theme.scss",
-                            "src/scss/styles.scss"
+                            "src/scss/styles.scss",
+                            "node_modules/@taiga-ui/core/styles/taiga-ui-global.less",
```

Comment:
Are sure this is ok? Do we support LESS?

Reply:
https://taiga-ui.dev/getting-started -> taigaUI getting started docs mentions angular automatically handles less files.
`You can use it with your pure CSS, SCSS and other projects. Angular resolves this less files automatically `